defaults/main.yml

```diff
@@ -1,4 +1,6 @@
 ---
 gluster_cluster_volume_base_path: /data/glusterfs
-gluster_cluster_storage_group_name: storage
+gluster_cluster_storage_group_name: cluster_storage
```

Comment:
I guess we used storage initially so as not to tie it to cluster-infra. We'd then create a storage group from the cluster_storage group using cluster-infra roles. Probably not an issue.